README.md

```diff
@@ -5,7 +5,20 @@
 This repository describes the conda package download data provided by Anaconda, Inc.  It includes package download counts starting from Jan. 2017 for the following download sources:
 
 * Anaconda Distribution: The default channels hosted on `repo.anaconda.com` (and historically on `repo.continuum.io`)
-* Select Anaconda.org channels: Currently this includes `conda-forge` and `bioconda`.
+  * `free`
+  * `main`
+* Select Anaconda.org channels, which has evolved over time (start dates indicated below):
+  * `anaconda`
+  * `bioconda`
+  * `conda-forge`
+  * `nvidia` - 2021-06
+  * `plotly` - 2021-09
+  * `pytorch` - 2023-01
+  * `pytorch-test` - 2023-01
```

Comment:
Just a small nit, not blocking but I think we _could_ communicate nicely with gte symbols here? As in:
```suggestion
  * `nvidia` >= 2021-06
  * `plotly` >= 2021-09
  * `pytorch` >= 2023-01
  * `pytorch-test` >= 2023-01
```